pkg/minuteman/plugin.go

```diff
@@ -7,10 +7,64 @@ import (
 	"log"
 	"os"
 
+	"github.com/containernetworking/cni/pkg/ns"
 	"github.com/containernetworking/cni/pkg/skel"
+
+	"github.com/vishvananda/netlink"
 )
 
 const DefaultPath = "/var/run/dcos/cni/l4lb"
+const IfName = "minuteman0"
```

Comment:
lets call the interface as "minuteman" and "spartan" just to be consistent with the host configuration else there would be unnecessary exceptions in the code